modules/core/include/opencv2/core/matx.hpp

```diff
@@ -57,6 +57,11 @@
 #include <initializer_list>
 #endif
 
+// WARNING: unreachable code using Ninja
+#if defined _MSC_VER && _MSC_VER >= 1928
+#pragma warning(disable: 4702)
+#endif
+
```

Comment:
Disabling in such way in a public header is a way similar to global disabling.

Could you provide link on build logs with these warnings?

Reply:
Yes, sure: https://github.com/opencv/opencv/runs/7454965078?check_suite_focus=true#step:13:584

Do you mean to disable each line with this warning?

Reply:
Removed the global definition and put to certain places which happen during the build.